wpical/src/main/native/cpp/fmap.cpp

```diff
@@ -22,23 +22,22 @@ std::string fmap::singleTag(int tag, tag::pose tagpose, bool endTag) {
       "{\"family\":\"apriltag3_36h11_classic\",\"id\":" + std::to_string(tag) +
       ",\"size\":165.1,\"transform\":[" + transform + "],\"unique\":true}";
 
-  if (!endTag) {
-    fmapTag += ",";
-  }
-  return fmapTag;
+  return wpi::json::parse(fmapTag);
```

Comment:
Don't parse it from a built string.. you can make the json as an object directly, e.g.:
```
return {{"family", "apriltag3_36h11_classic"},
          {"id", tag},
          {"size", 165.1},
          {"transform", std::move(transformjson)},
          {"unique", true}};
```
And you can dynamically build transformjson as a json array.